src/components/call_out/call_out.test.tsx

```diff
@@ -81,5 +82,35 @@ describe('OuiCallOut', () => {
         });
       });
     });
+
+    describe('dismissible', () => {
+      it('is rendered when set to true', () => {
+        const component = render(<OuiCallOut dismissible={true} />);
+
+        expect(component).toMatchSnapshot();
+      });
+
+      it('is not rendered when in warning color', () => {
+        const component = render(
+          <OuiCallOut dismissible={true} color={'warning'} />
+        );
+
+        expect(component).toMatchSnapshot();
+      });
+
+      it('is not rendered when in danger color', () => {
+        const component = render(
+          <OuiCallOut dismissible={true} color={'danger'} />
+        );
+
+        expect(component).toMatchSnapshot();
+      });
+
+      it('close callout after click', () => {
+        const component = mount(<OuiCallOut dismissible={true} />);
+        findTestSubject(component, 'closeCallOutButton').simulate('click');
+        expect(component).toMatchSnapshot();
```

Comment:
Are you sure this does what you expect? I took a look at the snapshot and cant really see a change here. Maybe take a snapshot before and after to show how the dismiss occurred.

Reply:
Add a snapshot before to show the difference.